docs/user-guide/import.md

```diff
@@ -16,10 +16,20 @@ Here the user, in order to import a file, can drag and drop it inside the import
 
 ## Import map context files
 
-A map context is, for example, the file that an user download selecting the <img src="../img/button/export.jpg" class="ms-docbutton"/> option in [Burger Menu](menu-bar.md#burger-menu). This type of files, in `json` format contains all map context information, that means the settings about projections, coordinates, zoom, extent and layers present in the map (more information can be found in the [Maps Configuration](../developer-guide/maps-configuration.md) section of the [Developer Guide](https://mapstore.readthedocs.io/en/latest/developer-guide/). Adding a map configuration file the behavior is similar to the following:
+A map context is, for example, the file that an user download selecting the <img src="../img/button/export.jpg" class="ms-docbutton"/> option in [Burger Menu](menu-bar.md#burger-menu). There are two types of file that can be exported:
```

Comment:
A map context is, for example, the file that an user download selecting the <img src="../img/button/export.jpg" class="ms-docbutton"/> option in [Burger Menu](menu-bar.md#burger-menu). Map contexts can be exported in two different format: